Adds Netstandard/Netframework support.

GitHubProUser67 opened the pull request:
Adds Netstandard and Netframework support.

Allows a wider range of usage such as Unity 2018 and Xamarin projects.

If possible it would be neat to update the Nuget like previously.

GitHubProUser67 committed:
Adds Netstandard2.1 support.

Adds Netstandard2.1 support.

Allows a wider range of usage such as Unity 2018 and Xamarin projects.

PJB3005 commented:
I am not sure this is worth the hassle. NS2.1 is already getting dated, and there are some useful future features (e.g. SNI support) that this would make impossible. I'm also not fond of the amount of extra code this adds to maintain.

GitHubProUser67 committed:
Implement NetStandard 2.0 support.

Implement NetStandard 2.0 support.

Also compatible with NetFramework 4.6.1 and up (maybe lower too).
